### PR TITLE
Update Kanban process with more info on PRs and other stuff

### PR DESCRIPTION
**Pull request by bartlettroscoe**
ToDo: Add comments/instructions here

### CODE REVIEW

**Review by mzuzek — COMMENTED** [review]
Hi @bartlettroscoe!

Is there anywhere a section explaining multiple projects and how are they created (e.g. for selected scope or specific time period) ?

**Comment by bartlettroscoe** [comment]
> Is there anywhere a section explaining multiple projects and how are they created (e.g. for selected scope or specific time period) ?

@MikolajZuzek, no.  The "TriBITS Refactoring" project is only going to live while there is significant funding for TriBITS refactoring work.  Once TriBITS goes back to maintenance mode and low-level development, the "TriBITS Refactoring" project will disappear and all work will go back to being tracked on the default "TriBITS" project board.

File: Kanban-Process-for-Issue-Tracking.md [hunk 1]
```diff
@@ -1,29 +1,53 @@
...
+## TriBITS GitHub Issue Process
+
+The intended Kanban process lifecycle for a **TriBITS GitHub Issue** is as follows (consistent with [TriBITS/CONTRIBUTING.md](https://github.com/TriBITSPub/TriBITS/blob/master/CONTRIBUTING.md)):
 
-The intended Kanban process lifecycle is as follows:
 * Someone creates a new Issue as a suggestion (and it automatically is placed in the Backlog).
 * The Product Owner then takes the Issue and reviews it in the Backlog and comments on it (or closes it if justified).
-* Regularly Issues are moved into "ToDo" or "Selected" when they are high enough priority and are ready to be worked.  (We should not cram a bunch of stuff into "Selected".  We should keep that list short so that it is clear what to work on next.)
-* Where adequate capacity is freed up, an Issue is moved from "Selected" to "In Progress" by the developer and it is actively worked.
+* Issues are regularly are moved into "ToDo" or "Selected" when they are high enough priority and are ready to be worked.  (We should not cram a bunch of stuff into "Selected".  We should keep that list short so that it is clear what to work on next.)
```
Comment (mperrinel) [inline Kanban-Process-for-Issue-Tracking.md:26]: Issues are regularly moved

File: Kanban-Process-for-Issue-Tracking.md [hunk 2]
```diff
@@ -1,29 +1,53 @@
...
-* Once the Issue (and associated pull request) is independently reviewed, then the branch is merged and pushed (if the Issue is for a code change) and the Issue is closed.
+* Once the Issue (and associated pull request) is independently reviewed, then the branch is merged and pushed (if the Issue is for a code change) and the Issue may be closed (if the PR completed resolved the issue).
 
-That is the basic Kanban process in a nutshell.
+That is the basic Kanban Issue process in a nutshell.  The process details are given in [TriBITS/CONTRIBUTING.md](https://github.com/TriBITSPub/TriBITS/blob/master/CONTRIBUTING.md).
 
-NOTES:
+## TriBITS Pull Request Process
+
+The Kanban lifecycle process fopr Pull Requests (PRs) follows a reduced number of Kanban stages:
```
Comment (mperrinel) [inline Kanban-Process-for-Issue-Tracking.md:36]: for Pull Requests

File: Kanban-Process-for-Issue-Tracking.md [hunk 3]
```diff
@@ -1,29 +1,53 @@
...
+
+* A new Pull Request is automatically added to the **In Progress** stage.
+* GitHub Actions testing is performed on each push to the PR branch with results displayed on [CDash](https://github.com/TriBITSPub/TriBITS/wiki/TriBITS-CDash-Dashboard).
+* When the PR is ready to be reviewed, it is moved to the **In Review** stage and reviewers are assigned.
+* Once the PR has been reviewed and accepted, it is merged (and the associated topic branch is merged to the main development branch).
```
Comment (mperrinel) [inline Kanban-Process-for-Issue-Tracking.md:41]: it is merged and moved to the **Done** stage

**Review by mperrinel — COMMENTED** [review]
Few typos found. I am creating the associated PR

**Comment by bartlettroscoe** [comment]
@mperrinel , thanks for the review.

Can you please post a PR with these changes, according to the instructions at:

* https://github.com/TriBITSPub/TriBITSWiki#readme

?

NOTE: I verified that 'master' in `git@github.com:TriBITSPub/TriBITS.wiki.git` is contained in 'master' in [`git@github.com:TriBITSPub/TriBITSWiki.git`](https://github.com/TriBITSPub/TriBITSWiki) so the PR will show the correct diff with just your changes.

**Comment by bartlettroscoe** [comment]
@mperrinel, note, you **must** create your topic branch off of 'master' in the production repo `git@github.com:TriBITSPub/TriBITS.wiki.git` and **not** the 'master' branch of the repo `git@github.com:TriBITSPub/TriBITSWiki.git`.

Just carefully follow the directions in the file [TriBITSWiki/README.md](https://github.com/TriBITSPub/TriBITSWiki/blob/master/README.md).

**Comment by mperrinel** [comment]
Ok, I'm on it

**Comment by mperrinel** [comment]
It is done. Let me know if I correctly followed the process.

**Comment by bartlettroscoe** [comment]
@mperrinel, did you post two identical PRs #2 and #3?  There are two different branch names and two different commits on these branches but the patches look identical.  Can you explain?

**Comment by mperrinel** [comment]
Sure. The #2 was not following the requirement you said 3 hours before.
The #3 has been done following your comments.
We can cancel the #2

**Comment by bartlettroscoe** [comment]
@mperrinel, you know you can force-push commits to a branch to GitHub?  So all you had to do was hard-reset your local branch, to cherry-pick your commit onto the updated branch and force push to the same remote branch in your fork and the PR #2 would have been updated to be correct.

I will close #2 and review and (manually) merge #3 as per the instructions.

**Comment by bartlettroscoe** [comment]
@mperrinel, I reviewed and (manually) merged PR #3 as per the documented process.  You can see the deployed change at:

* https://github.com/TriBITSPub/TriBITS/wiki/Kanban-Process-for-Issue-Tracking

which contains the changes from PR #3.

Thanks!

(This is the only what that I know to be able to use GitHub to use a PR to propose and formally review a change to a GitHub Wiki.)

File: Kanban-Process-for-Issue-Tracking.md (hunk 1 above)
Comment (bartlettroscoe) [reply Kanban-Process-for-Issue-Tracking.md:26]: Fixed in #3

File: Kanban-Process-for-Issue-Tracking.md (hunk 3 above)
Comment (bartlettroscoe) [reply Kanban-Process-for-Issue-Tracking.md:41]: Fixed in #3

File: Kanban-Process-for-Issue-Tracking.md (hunk 2 above)
Comment (bartlettroscoe) [reply Kanban-Process-for-Issue-Tracking.md:36]: Fixed in #3

**Pull request merged**